go/vt/vtctl/workflow/server.go

```diff
@@ -1289,14 +1293,24 @@ func (s *Server) MoveTablesComplete(ctx context.Context, req *vtctldatapb.MoveTa
 	if !state.WritesSwitched || len(state.ReplicaCellsNotSwitched) > 0 || len(state.RdonlyCellsNotSwitched) > 0 {
 		return nil, ErrWorkflowCompleteNotFullySwitched
 	}
-	var renameTable TableRemovalType
-	if req.RenameTables {
-		renameTable = RenameTable
+
+	if req.IgnoreSourceKeyspace {
```

Comment:
We should probably pass `req.IgnoreSourceKeyspace` into `dropSources()` because `dropSources()` is doing a few of things that are missed in the path where `req.IgnoreSourceKeyspace=true`:
* `validateWorkflowHasCompleted()`
* `dropTargetDeniedTables()`

Also we are locking the keyspace in the path for `req.IgnoreSourceKeyspace=false` in `dropSources()` and adding dry run results as well.

Reply:
Good point. This was the first change I made in this work and after having done everything else I completely agree. Initially I was thinking dropSources was only source side and dropTargets was only target side but that proved to be mistaken. I've made this change. Thanks!